ui/src/types/logEvent.ts

```diff
@@ -0,0 +1,3 @@
+import {LogEvent as Log} from 'src/client'
+
+export interface LogEvent extends Log {}
```

Comment:
I feel like this might be equivalent to just one line:
`export {LogEvent} from 'src/client'` ?

Reply:
That's a good point. I had originally written it like this in the event that other properties would need to be integrated but I'll go ahead and update it